media-libs/libshumate/libshumate-1.0.1.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org meson vala
+
+DESCRIPTION="Shumate is a GTK toolkit providing widgets for embedded maps"
+HOMEPAGE="https://wiki.gnome.org/Projects/libshumate"
+
+SLOT="0"
```

Comment:
I'm changing this to `SLOT="1.0/1"` since the shared object is libshumate-1.0.so.1.